core/services/ocr2/plugins/ccip/ccipexec/factory.go

```
@@ -111,6 +112,12 @@ func (rf *ExecutionReportingPluginFactory) NewReportingPluginFn(config types.Rep
 			return reportingPluginAndInfo{}, fmt.Errorf("get onchain config from offramp: %w", err)
 		}
 
+		msgVisibilityInterval := offchainConfig.MessageVisibilityInterval.Duration()
+		if msgVisibilityInterval < MinimumMessageVisibilityInterval {
+			msgVisibilityInterval = rf.msgVisibilityIntervalFallbackValue(onchainConfig.PermissionLessExecutionThresholdSeconds)
```

we should only fallback if `msgVisibilityInterval` is not set. If the property is set but less than minimum, it is an obvious misconfig, throwing an error feels more appropriate v.s using a default value that may be undesirable and hides the actual misconfig